base/cvd/cuttlefish/host/commands/cvd/parser/load_configs_parser.cpp

```diff
@@ -286,6 +309,7 @@ Result<CvdFlags> ParseCvdConfigs(const EnvironmentSpecification& launch,
   for (const auto& instance: launch.instances()) {
     flags.instance_names.push_back(instance.name());
   }
+  GenerateMissingInstanceNames(flags.instance_names);
```

Comment:
Can the behavior in lines 309-312 be part of `ParseLaunchCvdConfigs`?

Reply:
`ParseLaunchCvdConfigs` seems to be used to generate the flags for `cvd_internal_start` as a list of strings. The instance names are not passed to `cvd_internal_start` so I'd rather not mix those.